config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-periodics.yaml.tpl

```diff
@@ -216,3 +216,93 @@ periodics:
     testgrid-tab-name: capi-e2e-mink8s-{{ ReplaceAll $.branch "." "-" }}
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
+{{ if eq $.branch "release-1.4" "release-1.5" "release-1.6" | not }}
```

Comment:
❓ we want this job for future branches right (e.g. `release-1.7)`? If it is only there to be kept for main we should match for main.

Reply:
Yup!

Reply:
Just didn't want to backport